### PR TITLE
User list

### PR DESCRIPTION
**Pull request by YesicaNavaB**
## Summary

User list

## Trello Card

https://trello.com/c/01GM5yZA

### CODE REVIEW

File: app/controllers/users.js [hunk 1]
```diff
@@ -20,3 +20,12 @@ exports.signIn = (req, res, next) =>
       res.status(200).send({ token });
     })
     .catch(next);
+
+exports.userList = (req, res, next) =>
+  servicesUser
+    .userList(req.params)
+    .then(result => result)
```
Comment (jersonlopez) [inline app/controllers/users.js]: what is this `.then` doing?

File: app/controllers/users.js [hunk 2]
```diff
@@ -20,3 +20,12 @@ exports.signIn = (req, res, next) =>
...
+    .then(results => {
```
Comment (jersonlopez) [inline app/controllers/users.js]: remove brackets

File: app/services/users.js [hunk 3]
```diff
@@ -34,3 +34,22 @@ exports.signIn = ({ email, password }) =>
...
+    .catch(err => {
+      throw error.databaseError(err.message);
+    });
+};
+
+exports.validateToken = Authorization => {
+  const { email } = jwt.decode(Authorization, secret);
+  return User.findAndCountAll({ where: { email } }).then(result => {
+    if (result.count !== 1) {
+      throw error.databaseError('invalid Token ');
```
Comment (jersonlopez) [inline app/services/users.js]: if `result.count !== 1` is because the token is invalid, not a database error

**Review by jersonlopez — CHANGES_REQUESTED** [review]

File: app/controllers/users.js (hunk 1 above)
Comment (YesicaNavaB) [reply app/controllers/users.js]: ok!

File: app/controllers/users.js (hunk 2 above)
Comment (YesicaNavaB) [reply app/controllers/users.js]: ok!

File: app/services/users.js (hunk 3 above)
Comment (YesicaNavaB) [reply app/services/users.js]: perfect!

**Review by jersonlopez — APPROVED** [review]
good job! 🥇

**Pull request merged**